roles/storage-cinder/README.md

```diff
@@ -0,0 +1,87 @@
+# Storage Cinder Role
+
+This role aggregates Cinder, RabbitMQ and MariaDB to deploy Standalone
+Cinder with no authentication (noauth).
+
+MariaDB is deployed on a node with label=mariadb. MariaDB uses hostPath
+for storage.
+
+### Requirements
+
+
+### Role Variables
+| variable       | default           |choices           | comments  |
+|:-------------|:-------------|:----------|:----------|
+| action |  provision | <ul><li>provision</li><li>deprovision</li></ul>| Action to perform on the role|
+| infra_node_label | controller | | Label a node to allow MariaDB to utilize its hostpath
+| namespace | openstack | | A namespace where cinder and its dependencies will be deployed | 
+| service_account | cinder | | A service account with at least anyuid capability for use in OpenShift |
+| privileged_service_account | cinder_privileged | | A privileged service account for elevated privileges in OpenShift |
+| mariadb_root_password | weakpassword | | |
+| mariadb_user | root | | |
+| cinder_user | cinder | | |
+| cinder_password | cinderpassword | | |
+| rabbitmq_user | guest | | |
+| rabbitmq_password | rabbitmqpassword | | |
+
+### Backend-specific Variables
+
+#### Ceph
+| variable       | default           |choices           | comments  |
+|:-------------|:-------------|:----------|:----------|
+| enabled | false  | <ul><li>true</li><li>false</li></ul>| Enables Ceph backend |
+| cinder_rbd_pool_name | cinder_volumes  | | |
+| cinder_rbd_user_name | cinder  | | |
+| client_key | | | |
+| ceph_authentication_type | cephx  | | |
+| ceph_mon_host | | | IP address of Ceph Monitors. Comma-separated list of IPs |
+
+#### Xtremio
+| variable       | default           |choices           | comments  |
+|:-------------|:-------------|:----------|:----------|
+| enabled | false  | <ul><li>true</li><li>false</li></ul>| Enables Xtremio backend |
+| max_over_subscription_ratio | 40  | | |
+| use_multipath_for_image_xfer | | | |
+| volume_backend_name | xtremio | | |
+| san_ip | | | |
+| san_login | | | |
+| san_password | | | |
+| image_volume_cache_enabled | | | |
+
+#### NetApp
+| variable       | default           |choices           | comments  |
+|:-------------|:-------------|:----------|:----------|
+| enabled | false  | <ul><li>true</li><li>false</li></ul>| Enables Netapp backend |
```

Comment:
I mean for exampel netapp_enabled

Reply:
Hi, the 'enabled' var is part of a hierarchical dict, so it will look like netapp.enabled: true when used. I'd rather just use 'enabled' than netapp_enabled.